tutorial/README.rst

```diff
@@ -23,6 +23,12 @@ If you installed |MESSAGEix| using Anaconda, download the notebooks using the
 
     $ messageix-dl --local_path /path/to/tutorials
 
+.. note::
+
+   If you installed |MESSAGEix| using Anaconda, note that the version of
+   the tutorials that will be downloaded will depend on the version you are
+   currently in (it's indicated in green at the bottom left of this page).
```

Comment:
Sorry, I should have suggested to read the code—I wasn't familiar myself with what it says: https://github.com/iiasa/message_ix/blob/742fb899fa079708950e946d29e794c3df096787/message_ix/cli.py#L118-L130

So maybe revise the note box like this:
```rst
.. note::

   By default, the tutorials for your installed version of message_ix are downloaded.
   To download a different version, add e.g. ``--tag 1.2.0`` to the above command.
   To download the tutorials from the development version, add ``--branch master``.
```

Reply:
Thanks a lot Paul! I''l then replace the table with your Note. Indeed, I was not sure which version was Anaconda choosing to download the tutorials.

Reply:
Now I've included your changes @khaeru . I hope it's ok now! Pending of @volker-krey approval now I guess